trapper/commands.py

```diff
@@ -94,6 +103,90 @@ def add_subparser(
         return subparser
 
 
+@Subcommand.register("distributed-run")
+class DistributedRun(Subcommand):
+    """trapper's run command that enables running an experiment in
+    a distributed way.
+
+    Usage:
+        Basic:
+            ` trapper distributed-run experiment.jsonnet <--other-kwargs vals>`
+    """
+
+    def add_subparser(
+        self, parser: argparse._SubParsersAction
+    ) -> argparse.ArgumentParser:
+        description = """Distributed run the experiment specified in the config file which
+         specify the training and/or evaluation of a model on a dataset."""
+        subparser = parser.add_parser(
+            self.name,
+            description=description,
+            help="Start distributed training of a model.",
+        )
+
+        subparser.add_argument(
+            "config_path",
+            type=str,
+            help="path to the experiment config file in `json` or `jsonnet`"
+            " format describing the model, dataset and other details.",
+        )
+
+        subparser.set_defaults(func=torch_distributed_run)
+
+        return subparser
+
+
+def run_distributed(args):
+    """
+    Taken from https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py
```

Comment:
I suggest a specific tag or branch instead of `master` as the file content in the master might change with the new commits.